src/common/types.hpp

```diff
@@ -344,6 +376,14 @@ class Ip6Prefix
      */
     bool IsValid(void) const { return mLength > 0 && mLength <= 128; }
 
+    /**
+     * This method checks if the object is the default route prefix ("::/0")
+     *
+     * @returns true if the object is the default route prefix, false otherwise.
+     *
+     */
+    bool IsDefaultRoutePrefix(void) const { return (*this == Ip6Prefix("::", 0)); }
```

Comment:
What about we just check the prefix length is zero?

Reply:
both way should work good. but I prefer doing this as this is more common, so that similar method can be created using same way, e.g.
```
bool IsUlaPrefix(void) const { return (*this == Ip6Prefix("fc00::", 7)); }
```